other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -46,13 +46,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        await install_packages("create_interface_with_min_strain_zsl.ipynb","../../config.yml")
</pre><p>Are we keeping this change? If yes, then how will the packages be installed in Python (not Pyodide)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>